draft-ietf-quic-qpack.md

```diff
@@ -536,6 +536,31 @@ instruction is also processed as a Table Size Synchronize instruction with a
 value matching the Largest Reference of the corresponding header block.
 
 
+### Stream Cancellation
+
+A stream that is reset might have multiple outstanding header blocks.  A decoder
+that receives a stream reset before the end of a stream generates a Stream
+Cancellation instruction on the decoder stream.  Similarly, a decoder that
+abandons reading of a stream needs to signal this using the Stream Cancellation
+instruction.  This signals to the encoder that all references to the dynamic
+table on that stream are no longer outstanding.
+
+An encoder cannot infer from this acknowledgement that any dynamic table entries
+referenced have been received.
```

Comment:
I think you're trying to say that the encoder can't make any assumptions about the state of the dynamic table, as can be done with header acknowledgement.

"...cannot infer from this acknowledgement any updates to the dynamic table have been received" ?